pkg/operator/operator_config.go

```diff
@@ -20,12 +20,20 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	"sigs.k8s.io/controller-runtime/pkg/source"
+)
+
+// Base resource names which may be used for multiple different resource kinds
+// related to the given component.
+const (
+	NameOperatorConfig = "config"
+	NameRuleEvaluator  = "rule-evaluator"
+	NameCollector      = "collector"
 )
 
-// TODO(pintohutch): move these into the operator Options and pass in.
```

Comment:
I removed this TODO as in my opinion namespaces exist for precisely the reason to not have to make such things configurable whatsoever. If we allow arbitrary renaming of all resources we manage here, things can get really messy.